Document the shift in the criteria for inclusion of rules (#8194)
stylelint/stylelint: docs/about/vision.md

- [formatters](../developer-guide/formatters.md) - format Stylelint result objects
- [plugins](../developer-guide/plugins.md) - build custom rules to support methodologies, toolsets, non-standard CSS features, other languages (like SCSS) or very specific use cases

## Configurable

> Provide options to tailor the linter to specific project needs, including:
> 
> - [customizable rules](https://stylelint.io/user-guide/customize/) - adjust rules for specific scenarios or preferences
> - overlapping rules - allow users to choose between different approaches to validation

Reviewer: ```suggestion
Provide options to [customize the linter](../user-guide/customize.md) to specific project needs, including configurable rules that sometimes overlap.
```

To remove the redundancy.
